docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -108,11 +110,241 @@ Claims to monitor for size limits:
+          return redirect('/sign-in?redirect_url=' + args.request.url)
+        }
+
+        // Use the Backend SDK's `getUser()` method to get the Backend User object
+        const user = await createClerkClient({ secretKey: process.env.CLERK_SECRET_KEY }).users.getUser(
+          userId,
+        )
+
+        // Return the Backend User object
+        return { serialisedUser: JSON.stringify(user) }
+      }
+      ```
+
+      ```tsx {{ filename: 'routes/profile.tsx' }}
+      import { ActionFunction, redirect } from '@remix-run/node'
+      import { getAuth } from '@clerk/remix/ssr.server'
+      import { createClerkClient } from '@clerk/remix/api.server'
+
+      export const action: ActionFunction = async (args) => {
+        // Use `getAuth()` to get the user's ID
+        const { userId } = await getAuth(args)
+
+        // If there is no userId, then redirect to sign-in route
+        if (!userId) {
+          return redirect('/sign-in?redirect_url=' + args.request.url)
+        }
+
+        // Prepare the data for the mutation
+        const params = { firstName: 'John', lastName: 'Wicker' }
+
+        // // Use the Backend SDK's `updateUser()` method to update the Backend User object
+        const updatedUser = await createClerkClient({
+          secretKey: process.env.CLERK_SECRET_KEY,
+        }).users.updateUser(userId, params)
+
+        // Return the updated user
+        return { serialisedUser: JSON.stringify(updatedUser) }
+      }
+      ```
+    </CodeBlockTabs>
+  </Tab>
+
+  <Tab>
+    ```tsx {{ filename: 'app/routes/api/example.tsx' }}
+    import { createClerkClient } from '@clerk/backend'
+    import { json } from '@tanstack/react-start'
+    import { createAPIFileRoute } from '@tanstack/react-start/api'
+    import { getAuth } from '@clerk/tanstack-react-start/server'
+
+    export const Route = createAPIFileRoute('/api/example')({
+      GET: async ({ request, params }) => {
+        // Use `getAuth()` to get the user's ID
+        const { userId } = await getAuth(req)
+
+        // Protect the route by checking if the user is signed in
+        if (!userId) {
+          return json({ error: 'Unauthorized' }, { status: 401 })
+        }
+
+        // Instantiate the Backend SDK
+        const clerkClient = createClerkClient({ secretKey: import.meta.env.CLERK_SECRET_KEY })
+
+        // Use the Backend SDK's `getUser()` method to get the Backend User object
+        const user = userId ? await clerkClient.users.getUser(userId) : null
+
+        // Return the Backend User object
+        return json({ user })
+      },
+    })
+    ```
+  </Tab>
+</Tabs>
+
+However, if you make this call to Clerk's Backend API frequently, you risk hitting [rate limits](/docs/backend-requests/resources/rate-limits) and it's also slower than making a database query. So it's recommended to [store the extra data in your own database](/docs/webhooks/sync-data#storing-extra-user-data) instead of storing it in metadata in the session token.
```

Comment:
It seems weird to recommend making a BAPI call, put a bunch of examples of how to do it, then immediately after being like "actually, we recommend not doing this". I feel like I'd flip these around -- explicitly recommend storing larger user data in your own database, then mention that if there's some restriction that makes this impossible, it is possible to fetch via BAPI, but risky because of rate limits, so we do not recommend this.